source/common/router/route_config_update_receiver_impl.h

```diff
@@ -42,9 +42,7 @@ class RouteConfigUpdateReceiverImpl : public RouteConfigUpdateReceiver {
       : config_traits_(optional_http_filters,
                        factory_context.messageValidationContext().dynamicValidationVisitor()),
         base_(config_traits_, proto_traits, factory_context), last_vhds_config_hash_(0ul),
-        vhds_virtual_hosts_(
-            std::make_unique<std::map<std::string, envoy::config::route::v3::VirtualHost>>()),
-        vhds_configuration_changed_(true) {}
+        vhds_virtual_hosts_(nullptr), vhds_configuration_changed_(true) {}
```

Comment:
you can omit nullptr inits for unique_ptr.

Reply:
Done